lib/components/output-area.js

```diff
@@ -9,10 +9,19 @@ import { OUTPUT_AREA_URI } from "./../utils";
 
 import typeof store from "../store";
 
+function EmptyMessage() {
+  return (
+    <div>
+      <ul className="background-message centered">
+        <li>No output to display</li>
+      </ul>
+    </div>
```

Comment:
I assumed so because the same no output message below is wrapped in a div. I suppose it is unnecessary here now that I think about it.